README.md

```diff
@@ -28,7 +29,6 @@ This plugin adds Go language support for Vim, with the following main features:
   errors, or make sure errors are checked with `:GoErrCheck`.
 * Advanced source analysis tools utilizing `guru`, such as `:GoImplements`,
   `:GoCallees`, and `:GoReferrers`.
-* Precise type-safe renaming of identifiers with `:GoRename`.
```

Comment:
Is this removed accidentally?

Reply:
If I remember correctly, at some point it was decided to keep the top 10 or 15 points from the help file here instead of all. So I removed the bottom one after I added the debug feature.

Reply:
Can we remove the `automatic gopath` detection line and keep this? I think the GOPATH one can change as it's less relevant nowadays.

Reply:
Sure, sounds good